zb-classifier.js

```diff
@@ -499,13 +517,16 @@ class ZigbeeClassifier {
     return property;
   }
 
-  addButtonLevelProperty(node, genLevelCtrlOutputEndpoint) {
+  addButtonLevelProperty(node, genLevelCtrlOutputEndpoint, suffix) {
+    if (typeof suffix === 'undefined') {
+      suffix = '';
+    }
     const property = this.addProperty(
       node,                           // device
-      'level',                        // name
+      `level${suffix}`,                        // name
```

Comment:
nit: comment spacing should align with previous line

Reply:
done